### PR TITLE
Fastlane strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by dennisguse** [comment]
@pstorch @mondlicht-und-sterne @rgmf What are you opinions?

**Comment by pstorch** [comment]
I'm still thinking about the "buddy" word. It sounds a little bit unusual, but I'm not a native speaker.

**Comment by rgmf** [comment]
I like the new description, specially the style of saying what OpenTracks does. But English is not my mother tongue so I can't help on the details.

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
-* Photos and markers: mark interesting locations while tracking
-* Altitude gain and loss: via barometric sensor
-* Altitude shown in EGM2008 (above mean sea level); exported as WGS84
-* Voice announcements
-* Supports Bluetooth LE sensors: heart rate, speed and distance (cycling), cadence (cycling), and power meter (cycling)
-* Export data: export tracks either as KMZ (incl. photos), KML, or GPX
-* No Internet access: Internet is not used
+A sport tracking buddy that respects your privacy.
+
+sport and outdoor activities grants happiness, skill and confidence.
```
Comment (rgmf) [inline fastlane/metadata/android/en-US/full_description.txt:3]: This paragraph must start with capitalized S leeter, right?

**Review by rgmf — CHANGES_REQUESTED** [review]

File: fastlane/metadata/android/en-US/full_description.txt [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+* Altitude gain and loss: via barometric sensor.
+* Altitude shown in EGM2008 (above mean sea level); exported as WGS84.
+* Export data as tracks either as KMZ (incl. photos), KML, or GPX.
+* No Internet access or extra permissions.
+* Dark and light theme, respecting system settings.
+* No ads.
+
+Libre software / Free Software / Open Source
+meaning you can use, study, change and share the source code.
+https://github.com/OpenTracksApp/OpenTracks
```
Comment (dennisguse) [inline fastlane/metadata/android/en-US/full_description.txt:22]: Why put the link here?
It is shown in F-Droid anyhow below.

**Review by dennisguse — APPROVED** [review]
I like the changes.

File: fastlane/metadata/android/en-US/full_description.txt (hunk 2 above)
Comment (comradekingu) [reply fastlane/metadata/android/en-US/full_description.txt:22]: I was thinking it was better for the Play listing, but it is available at the bottom there too.
It doesn't work for searches either, so removing :)

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
Comment (comradekingu) [reply fastlane/metadata/android/en-US/full_description.txt:3]: Good catch :)

**Comment by dennisguse** [comment]
@comradekingu Could you squash the commits?

**Pull request merged**